openpathsampling/netcdfplus/chaindict.py

```diff
@@ -57,6 +57,8 @@ class ChainDict(object):
 
     def __init__(self):
         self._post = None
+        self._iterables = (list, tuple)
```

Comment:
should `set` be included as an iterable?

Reply:
yes. The check for __iter__ or Collections.iterables is better but much slower. `set`, `frozenset` make sense though.